keras_contrib/backend/tensorflow_backend.py

```diff
@@ -158,3 +160,22 @@ def moments(x, axes, shift=None, keep_dims=False):
     ''' Wrapper over tensorflow backend call '''
 
     return tf.nn.moments(x, axes, shift=shift, keep_dims=keep_dims)
+
+
+def clip(x, min_value, max_value):
```

Comment:
This was modified because the `K.clip` in Keras doesn't allow tensors for `min_value` or `max_value`.

Reply:
Documentation needs to be updated accordingly. Could these changes be valuable in keras itself?

Reply:
@ahundt Updated docstrings for all 3 backends. I think these changes could be useful in Keras for people writing new layers.